pyproject.toml

```diff
@@ -153,6 +153,7 @@ classmethod-decorators = ["classmethod", "validator", "root_validator", "pydanti
     "D103",
     "D100",
 ]
+"tests/integration/*" = ["F401", "F811"]
```

Comment:
Perhaps we could ignore this on a per-line basis with an in-line: "# noqa: F401"? Ruff cleaning up unused imports on save is kind of nice, in my opinion.

Reply:
The reason we added it was because it reports imported fixtures as unused imports. And since we have sentralized shared fixtures in `utils`, we would have to ignore F401 for all theses imports in every file and every future file.